site/content/blog/2016-07-19-community-building.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
so we would need to port over all the remaining blog posts on medium

Reply:
yep, this could be in a future PR